### PR TITLE
Feature/evidence summary handler

### PR DESCRIPTION
**Pull request by JohnPadgettCivica**
Just implementation of the initial query and tests

### CODE REVIEW

File: src/EA.Weee.Tests.Core/NoteCommon.cs [hunk 1]
```diff
@@ -7,7 +7,9 @@
     using System.Threading.Tasks;
     using Domain.AatfReturn;
     using Domain.Evidence;
+    using Domain.Lookup;
```
Comment (NateHogan) [inline src/EA.Weee.Tests.Core/NoteCommon.cs:10]: Unused imports

File: src/EA.Weee.Database/scripts/Update/Release 5/Sprint03/20220420-1439-totals-stored-proc.sql [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+SET NOCOUNT ON;
+
+IF OBJECT_ID('tempdb..#CategoryTotals') IS NOT NULL 
+BEGIN
+	DROP TABLE #CategoryTotals
+END
+
+CREATE TABLE #CategoryTotals(
+	CategoryId INT NOT NULL,
+	CategoryName NVARCHAR(500) NOT NULL,
```
Comment (NateHogan) [inline src/EA.Weee.Database/scripts/Update/Release 5/Sprint03/20220420-1439-totals-stored-proc.sql:18]: Although fairly minor, the datatype for [Lookup].[WeeeCategory].CategoryName is nvarchar(60), is the 500 just for safety?

File: src/EA.Weee.DataAccess.Tests.DataAccess/StoredProcedure/GetAatfEvidenceSummaryTotalsTests.cs [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+    {
+        private readonly Fixture fixture;
+
+        public GetAatfEvidenceSummaryTotalsTests()
+        {
+            fixture = new Fixture();
+        }
+
+        [Fact]
+        public async Task Execute_GivenAatfWithDraftNote_NoDataShouldBeReturned()
```
Comment (NateHogan) [inline src/EA.Weee.DataAccess.Tests.DataAccess/StoredProcedure/GetAatfEvidenceSummaryTotalsTests.cs:29]: Could the first three tests be replaced with a [Theory] With something like "GivenAatfWithNonApprovedNote_NoDataShouldBeReturned(NoteStatus status)"

Comment (JohnPadgettCivica) [reply src/EA.Weee.DataAccess.Tests.DataAccess/StoredProcedure/GetAatfEvidenceSummaryTotalsTests.cs]: Good idea, have updated.

File: src/EA.Weee.Database/scripts/Update/Release 5/Sprint03/20220420-1439-totals-stored-proc.sql (hunk 2 above)
Comment (JohnPadgettCivica) [reply src/EA.Weee.Database/scripts/Update/Release 5/Sprint03/20220420-1439-totals-stored-proc.sql]: Good spot, updated.

File: src/EA.Weee.Tests.Core/NoteCommon.cs (hunk 1 above)
Comment (JohnPadgettCivica) [reply src/EA.Weee.Tests.Core/NoteCommon.cs]: removed.

**Review by NateHogan — APPROVED** [review]

**Pull request merged**